docs/en/operation/Basic-Logging.md

```diff
@@ -1,43 +1,68 @@
 ---
 layout: global
-title: Server Logging
-nickname: Server Logging
+title: Basic Logging
+nickname: Basic Logging
 group: Operations
-priority: 12
+priority: 11
 ---
 
 * Table of Contents
 {:toc}
 
 ## Introduction
 
-This page summarizes Alluxio's logging system for servers, such as the master and worker
-processes. We include tips for modifying Alluxio's log4j properties file to best suit
-deployment needs. If you are a user of Alluxio looking for logging information about clients
-that are utilizing the Alluxio API, we recommend looking at the
-[client logging documentation]({{ '/en/operation/Client-Logging.html' | relativize_url }})
-
-Alluxio's logging behavior can be fully configured through the `log4j.properties` file found in the
-`conf` folder.
+This page describes the basic logging function provided by Alluxio server processes (e.g., masters, workers and etc)
+and application processes utilizing Alluxio clients (e.g., Spark or MapReduce jobs running on Alluxio).
+Note that, there is an advanced feature that streams server-side and client-side logs to separate Alluxio logservers
```

Comment:
no comma -> `Note that there is an advanced...`

Reply:
ping

Reply:
done